docs/SETUP.md

```
@@ -1,243 +1,73 @@
-Pick one method. The shell installer is recommended — it downloads the correct binary for your platform and runs `mag setup` automatically.
-
-```bash
-curl -fsSL https://raw.githubusercontent.com/George-RD/mag/main/install.sh | sh
-```
-
-If you prefer a package manager:
+Choose one method:
 
 | Method | Command |
 |---|---|
+| Shell | `curl -fsSL https://raw.githubusercontent.com/George-RD/mag/main/install.sh | sh` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)